src/components/NFTcard/nftcard.tsx

```diff
@@ -12,11 +14,15 @@ export interface NftCardProps {
 }
 
 const NTFCard: React.FC<NftCardProps> = ({ cardContent, navigateToCard, addToWhiteList, placeBid }) => {
+  const { t } = useTranslation()
+
+  const imgUrl = uriToHttp(`${process.env.REACT_APP_IPFS_URL}${cardContent.image}`)[0]
```

Comment:
remove IPFS url
its not working

Reply:
its working 
![image](https://user-images.githubusercontent.com/23531140/121229843-c58a0480-c88e-11eb-9327-d4c7016d15eb.png)